### DISCUSSION TITLE
How to show label the y-axis?

### DISCUSSION THREAD

**Discussion by xzrka** [highfestiva/finplot#394] [Q&A]
![ccc](https://user-images.githubusercontent.com/118905121/209737271-5ffd6a59-9c66-4713-84e2-b746f66ad7c6.png)
This is the chart I drew using your module

<img width="322" alt="dsdsdsdsdddd" src="https://user-images.githubusercontent.com/118905121/209737157-5a459ede-e1dc-4bbf-aca7-a2015085fb86.PNG">
This is a chart from TradingView

First of all, I didn't know how to display it on the y-axis, so I made it like that.
How to display labels on y-axis like in trading view below?

**Comment by highfestiva** [comment]
Currently, there unfortunately is no such functionality in finplot. How did you add the text shown in your first screenshot? I suppose you could `.addItem()` to the Y-axis and then move that around. If you think of a neat way to do it that would fit in the library, please send a pull request.

**Reply by xzrka** [reply]
last_price = df.iloc[113]['close']
    yesterday_price = df.iloc[112]['close']
    

    label_opts = {'position': 0.953, 'color': (255, 255, 255),
                      'fill': (200, 200, 200, 50), 'movable': True}
    
    ax.price_line = pg.InfiniteLine(angle=0, movable=False, pen=fplt._makepen(fplt.candle_bull_body_color, style='*'), label=str(format(last_price, ",.12g")),labelOpts=label_opts)
    
    #ax.price_line.pg.InfiniteLine.labelOpts(ax.price_line, text='', movable=False, position=0.5)
    
    ax.price_line.addMarker('v', position=1, size=15.0)
    
    print("df.iloc[113]['close'] : ",type(last_price), ',    ',last_price)
    
    x = str(last_price)
    x = float(x)
    
    u_price_line_color = fplt.volume_bull_color
    
    if last_price >= yesterday_price:
        u_price_line_color = fplt.volume_bull_color
        
    elif last_price <= yesterday_price:
        u_price_line_color = fplt.volume_bear_color
            
    print("x = ", type(x), x)
    if float(x) > 1000000:
        print("pos 재조정")
        ax.price_line.setPos(last_price * 0.000001)
    else:
        ax.price_line.setPos(last_price)
        
    print(u_price_line_color)
    ax.price_line.pen.setColor(pg.mkColor(u_price_line_color))
    ax.addItem(ax.price_line, ignoreBounds=True)

    #크로스헤어, x축, y축, x그리드, y그리드
    fplt._ax_set_visible(ax, False, False, True, True, True)
    fplt._ax_set_visible(axv, False, True, True, True, True)
    
    hover_label = fplt.add_legend('', ax=ax)
    rawtxt = '<span style="font-size:22px;color:rgb(255, 255, 255);">' + 'Binance ' + symbol.upper() + ' ' + textinteval+ '</span> &nbsp;' 
    hover_label.setText(rawtxt )
    
    fplt.legend_border_color = '#1e262f'
    fplt.legend_fill_color = '#1e262f'
    fplt.legend_text_color = '#ffffff'
    #fplt.add_legend('ㅤVOLUME', ax=axv)
    
    volume_label = fplt.add_legend('', ax=axv)
    vrawtxt = '<span style="font-size:13px;color:rgb(255, 255, 255);text-align:center;">ㅤVOLUME</span> &nbsp;' 
    volume_label.setText(vrawtxt )
    

----------------------------------------------------------
    The text displayed in the screenshot is this.
    
    And if there is a way to move to Y-axis, please show me with example code.

**Reply by highfestiva** [reply]
Unfortunately I'm not able to come up with a simple solution. It might require some new features in the way axes work. Let me know if you come up with something.

**Reply by xzrka** [reply]
Thanks for your reply. If anything comes to mind, I will respond.
